GliaWidgets/SecureConversations/SecureConversations.Coordinator.swift

```diff
@@ -46,6 +46,39 @@ extension SecureConversations {
 
             return controller
         }
+
+        private func makeSecureConversationsConfirmationViewController() -> SecureConversations.ConfirmationViewController {
+            let viewModel = SecureConversations.ConfirmationViewModel(
+                environment: .init(
+                    confirmationStyle: viewFactory.theme.secureConversationsConfirmationStyle
+                )
+            )
+
+            let controller = SecureConversations.ConfirmationViewController(
+                viewFactory: viewFactory,
+                props: viewModel.props()
+            )
+
+            viewModel.delegate = { [weak self, weak controller] event in
+                switch event {
+                case .backTapped:
+                    self?.delegate?(.backTapped)
+                case .closeTapped:
+                    self?.delegate?(.closeTapped)
+                // Bind changes in view model to view controller.
+                case let .renderProps(props):
+                    controller?.props = props
+                }
+            }
+
+            // Store view model, so that it would not be deallocated.
+            // Currently the view model can only be of one type because
+            // the base `ViewModel` class has associated values. Will need
+            // to fix this in the future.
+            // self.viewModel = viewModel
```

Comment:
didn't catch the comment, so do we need to have this line or it could be removed already?

Reply:
The view model is stored here so that it doesn't get deallocated. However, right now this view model is of type `SecureConversations.WelcomeViewModel`. So  right now this line has to be commented, but at some point, it should not. I decided to keep it here to remember how it should work, rather than just have a comment but no indication on what should be the end result. Don't know if that is the right approach.